src/containers/Masthead/components/MastheadSearch.tsx

```diff
@@ -302,6 +312,25 @@ const MastheadSearch = () => {
                 <SearchLine />
               </IconButton>
             </ComboboxControl>
+            <StyledHitsWrapper aria-live="assertive">
+              {!loading && query && (
+                <div>
+                  {!(mappedItems.length > 1) ? (
+                    <Text textStyle="label.small">{t("searchPage.noHitsShort", { query: query })}</Text>
+                  ) : (
+                    <Text textStyle="label.small">{`${t("searchPage.resultType.showingSearchPhrase")}${query}`}</Text>
```

Comment:
Må i det minste ha et mellomrom her.
```suggestion
                    <Text textStyle="label.small">{`${t("searchPage.resultType.showingSearchPhrase")} ${query}`}</Text>
```